src/containers/LicenseRevokeModal/LicenseRevokeModal.test.jsx

```diff
@@ -6,16 +6,17 @@ import configureMockStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 import { mount } from 'enzyme';
 
-import * as licenseService from '../../components/subscriptions/data/service';
+import LicenseManagerApiService from '../../data/services/LicenseManagerApiService';
```

Comment:
has it been decided to shift the subscription service to the new location?

Reply:
We have other ApiServices in one place here. To keep it generic and easy to trace it has been moved here. Should we keep it here or move it back in subscriptions?

Reply:
@adamstankiewicz Can you respond on the above?

Reply:
The organization of files/folders in the admin portal have followed the container/component structure since that was "best practice" at the time of creating the initial app. 

However, most MFEs are no longer using that structure in favor of keeping things organized by the app's functionality, not by how the implementation details of how the app is built (e.g., splitting up containers/components). 

That way, components become more like self-contained modules that contain everything they need all in one place, as opposed to having related files scattered throughout the codebase. See https://github.com/edx/frontend-app-course-authoring/blob/master/docs/decisions/0002-feature-based-application-organization.rst for example.

This is the pattern the Enterprise Learner Portal follows, and the "Subscriptions" page within the admin portal was the first attempt at starting to use this feature-based organization approach within the repo.

That said, I would prefer if we tried to keep anything subscriptions related under the "subscriptions" folder so it is self-contained. I know we've strayed from this already, though... 🤷‍♂️